packages/itwinui-css/src/tag/tag.scss

```
@@ -15,29 +14,28 @@
   @include iui-tag-base;
 
   &:not(a) {
-    color: var(--iui-text-color-muted);
+    color: var(--iui-color-foreground-4);
   }
 }
 
 @mixin iui-tag {
   @include iui-tag-base;
-  height: $iui-baseline * 3;
-  margin-top: $iui-component-offset;
-  margin-bottom: $iui-component-offset;
-  border-radius: $iui-border-radius-round;
-  padding: 0 $iui-xxs;
+  height: calc(var(--iui-size-s) * 3);
```

```suggestion
  --_iui-tag-height: var(--iui-component-height);
  
  height: var(--_iui-tag-height);
```

Our tags should probably use the new component height variable.

---

I don't think tags need to always have the same height as inputs/buttons (especially since they have different shapes), so this variable wouldn't make sense here.